### PR TITLE
ci(e2e): improved e2e snapshotting

### PR DESCRIPTION
**Pull request by inc-man**
# Motivation

Our E2E Tests often fail due to flakiness or instability of snapshots. With these added features we will ensure better stability for snapshots.

# Changes

added functionality to snapshotting

# Tests

ran workflows successfully

### CODE REVIEW

File: playwright.config.ts [hunk 1]
```diff
@@ -38,9 +46,14 @@ export default defineConfig({
...
 	projects: [
+		/* Test against desktop browsers. */
 		{
 			name: 'Google Chrome',
-			use: { ...devices['Desktop Chrome'], channel: 'chrome' }
+			use: { ...devices['Desktop Chrome'] }
+		},
+		{
+			name: 'Firefox',
+			use: { ...devices['Desktop Firefox'] }
```
Comment (AntonioVentilii) [inline playwright.config.ts:56]: this in another PR right?

**Review by AntonioVentilii — COMMENTED** [review]
Shall we create a generic method inside Homepage so that we avoid repetition of code?

```typescript
// pseudo code
toHaveScreenshotWithTimeout = async (
  options: Parameters<typeof page.toHaveScreenshot>[0] = {}
) => {
  return await self.#page.toHaveScreenshot({
    ...options,
    timeout: 1000 * 60,
  });
};
```

File: playwright.config.ts (hunk 1 above)
Comment (inc-man) [reply playwright.config.ts:56]: absolutely, that was a copy mistake I did from the other PR. will be removed

File: playwright.config.ts [hunk 2]
```diff
@@ -22,6 +22,14 @@ const TIMEOUT = 5 * 60 * 1000;
 export default defineConfig({
 	timeout: TIMEOUT,
 	workers: DEV ? 5 : 2,
+	expect: {
+		toHaveScreenshot: {
+			threshold: 0.25,
+			maxDiffPixelRatio: 0.025,
+			animations: 'disabled',
+			caret: 'hide'
```
Comment (AntonioVentilii) [inline playwright.config.ts:30]: as discussed offline, let's not make the tests more lax, let's keep them as they are and focus to check where and how the differences arises

**Comment by inc-man** [comment]
superseded by https://github.com/dfinity/oisy-wallet/pull/4019

**Pull request closed without merging**